tracer/src/Datadog.Trace/AppSec/AppSecRequestContext.cs

```diff
@@ -1,22 +1,27 @@
-﻿// <copyright file="AppSecRequestContext.cs" company="Datadog">
+// <copyright file="AppSecRequestContext.cs" company="Datadog">
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
 #nullable enable
 
+using System;
 using System.Collections.Generic;
+using Datadog.Trace.AppSec.Rasp;
 using Datadog.Trace.Tagging;
 using Datadog.Trace.Vendors.Newtonsoft.Json;
 
 namespace Datadog.Trace.AppSec;
 
 internal class AppSecRequestContext
 {
+    private const string _stackKey = "_dd.stack";
+    private const string _exploitStackKey = "exploit";
     private readonly object _sync = new();
     private readonly List<object> _wafSecurityEvents = new();
+    private Dictionary<string, List<Dictionary<string, object>>>? _raspStackTraces = null;
 
-    internal void CloseWebSpan(TraceTagCollection tags)
+    internal void CloseWebSpan(TraceTagCollection tags, Span span)
```

Comment:
wondering if we should stop passing the `tags` alltogether and just access them l.31 with `span.SetTag`